library/rtsp/src/main/java/com/google/android/exoplayer2/source/rtsp/MediaDescription.java

```diff
@@ -314,7 +314,8 @@ public int hashCode() {
 
     // Format of the parameter: RFC3640 Section 4.4.1:
     //   <parameter name>=<value>[; <parameter name>=<value>].
-    String[] parameters = Util.split(fmtpComponents[1], ";\\s?");
+    // Split with implicit limit of 0 to handle an optional trailing semicolon.
+    String[] parameters = fmtpComponents[1].split(";\\s?");
```

Comment:
Maybe use an explicit limit, like

`fmtpComponents[1].split(";\\s?", /* limit= */ 0);`

Reply:
No problem. It'll be [functionally equivalent](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#split-java.lang.String-), but I'm happy to make the change.